pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## What's the purpose of this pull request?
+<em>Considering the context, what is the problem we'll solve? Where in VTEX's big picture our issue fits in? Write a tweet about the context and the problem itself.</em>
+
+## How it works? 
+<em>Tell us the role of the new feature, or component, in its context.</em>
+
+## How to test it?
+<em>Describe the steps with bullet points. Is there any external reference, link, or example?</em> 
```

Comment:
What would be the difference between the reference requested here and the one in "References" section? 
It might be better to use another nomenclature or explain it so the dev don't get confused as if we're asking for the same thing in 2 different sections